tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "incremental": true,
     "baseUrl": ".",
     "outDir": "dist",
-    "rootDir": "packages",
+    "rootDir": ".",
```

Comment:
This was needed because Jest complained about not having access to `@cloudflare/workers-types` which exists in `node_modules` in the actual root directory. This is likely because the built version of this package has .ts files which jest thinks it needs to compile